designs/README.md

```diff
@@ -640,15 +692,38 @@ based designs details with following order:
          Project
     -   Executable CI Script: [rd_yifive](../scripts/rd_yifive)
 
+-   `mpw5_riscduino_dcore`
+    -   Project Name: Riscduino-DCore
+    -   Project Owner: Dinesh Annaya
+    -   Project Number: [718](https://platform.efabless.com/projects/718)
+    -   Description: Riscduino is a Dual 32 bit RISC V based SOC design
+         pin compatible to arudino platform and this soc targeted for
+         efabless Shuttle program.
+    -   Executable CI Script: [rdd_ycr2_iconnect](../scripts/rdd_ycr2_iconnect)
+
 -   `riscduino_dcore`
-    -   Project Name: riscduino-DCore
+    -   Project Name: Riscduino-DCore
     -   Project Owner: Dinesh Annaya
     -   Project Number: [838](https://platform.efabless.com/projects/838)
     -   Description: Riscduino is a Dual 32 bit RISC V based SOC design
          pin compatible to arudino platform and this soc targeted for
          efabless Shuttle program.
     -   Executable CI Script: [rdd_ycr2_iconnect](../scripts/rdd_ycr2_iconnect)
 
+-   `mpw5_riscduino_qcore`
+    -   Project Name: Riscduino-QCore
+    -   Project Owner: Dinesh Annaya
+    -   Project Number: [782](https://platform.efabless.com/projects/782)
+    -   Description: Riscduino is a Dual 32 bit RISC V based SOC design
+         pin compatible to arudino platform and this soc targeted for
+         efabless Shuttle program.
+    -   Executable CI Script: [rdq_uart_i2c_usb_spi_top](../scripts/rdq_uart_i2c_usb_spi_top) |
```

Comment:
`rdq_uart_i2c_usb_spi_top` vs `rdq_uart_i2cm_usb_spi_top`
The second is the correct name for the script folder